src/asmcnc/apps/systemTools_app/screens/screen_factory_settings.py

```diff
@@ -690,9 +690,8 @@ def remove_csv_files(self):
             pass
 
     def factory_reset(self):
-
         def nested_factory_reset():
-
+            self.m.disable_ssh()
```

Comment:
this should happen later in the factory reset, in case there are errors and we need to access the machine (ESPECIALLY git fsck errors)